setup.py

```diff
@@ -37,6 +37,10 @@ def run(self):
         Extension(
             "IM_calculation.IM.rspectra_calculations.rspectra",
             ["IM_calculation/IM/rspectra_calculations/rspectra.pyx"],
+        ),
+        Extension(
```

Comment:
If you use pip does it automatically build the two cython-y bits?

Reply:
Yeah, it should build everything in the list